### ISSUE TITLE
Imagine a place !

### ISSUE THREAD

**Issue by FaustinM** [UpsilonNumworks/Upsilon#91]
TL;DR : We need to create a new place for Upsilon's discussion, made by and for Upsilon's contributors

# Introduction
During the creation of Omega, we need to have a way to discuss together. A discord's server was created by the maintainers of Omega (Quentin, M4x1m3 and RedGl0w). The discord is design to respect theirs rules and convictions.

After archiving of Omega, the Discord got rename to Omega Community and open to new projects around Numworks and Epsilon.

After some discussion with Quentin, we obtain a unique channel to talk about Upsilon.

# Why we need to move
Actually, the server is manage by older lead team. But the old team don't contribute at all in Upsilon. We don't have any PR or interaction in this repo from any member of the old core team! 

Upsilon represent more than one message out of six in the discord's server, but Laury, Upsilon's leader, don't have any power.
So without any contribution, the older lead team, can add any mod, can delete our channel without warning, can delete message or censor anything. It's critical because all of our debate history is on Discord.

Recently, Quentin, Omega Community owner, won't edit the welcome warning, by this action, he had power and right on a project without any legitimacy outside the fact he own our communication platform.

Obviously, with the time and the new website, the promotion of Omega Community is made by the contributors of Upsilon and theirs actions but the Upsilon's contributors can't vote and manage the server.

Yes we will divide the community, but with the new website and the popularity of Upsilon, we will keep our user base, and we will leave who don't care about our fork

# How to move
Yes, we need a discord. A lot of our users don't know a lot about GitHub and the way GitHub is made is not usable for instant discussion. 

But we require a discord made by and for the contributors of Upsilon. We can follow the example of [NotAnName](https://github.com/NotANameServer/discord), they use a GitHub for the rules and the settings of the Discord. It's more transparent and democratic.

We can use our Omega channel, Numworks's Reddit and a lot of other way to promote the new way to chat at Upsilon project

**Comment by FaustinM** [comment]
I didn't mention that but if we have a dedicated server for Upsilon we can obtain some [advantage and promotion](https://discord.com/open-source) from Discord, actually we can't because Omega Community is not only about our Open Source project.

**Comment by quentinguidee** [comment]
The idea could have been better brought up. With such a message, you, and Uspilon by extension, make the decision to go against Omega. So be it.
It's one way to go, but it would have been smarter to just talk about a new platform without creating a drama.

> […] but Laury, Upsilon's leader, don't have any power.

The request was not made.

> So without any contribution, the older lead team, can add any mod, can delete our channel without warning, can delete message or censor anything. It's critical because all of our debate history is on Discord.

Yes we *can*, but if you think about it for 30 seconds it's not in anyone's best interest to delete the entire channel like that, censor a discussion about Upsilon, or deprive yourself of history. We are human and have no interest in doing anything quoted here.

> Recently, Quentin, Omega Community owner, won't edit the welcome warning, […]

You have intentionally truncated this sentence. The message has been edited and is much smoother now, democracy wanted this in-between. Afterwards, your opinion as a unitary person, even in a 100% Upsilon server, will not prevail. It will take some agreement between parties.

> […] by this action, he had power and right on a project without any legitimacy outside the fact he own our communication platform.

In addition, as you have noticed, in the very principle of Discord roles a certain hierarchy means that not everyone has the right to delete or modify everything. You didn't have to establish a rigorous demonstration to realize this, and this is the case with a rather high number of Discord servers :)

**Comment by FaustinM** [comment]
Yes, I make the decision to open a debate here. We can't talk about this problematic in a platform where some of us have more power than others. Who can ensure you don't censor our debate, or try to alter the reality ?
So it's just more fair for everyone to talk about that in a platform with more transparency.

Btw, [this message](https://i.imgur.com/EaqxwxV.png) explains why I can't talk without any trouble.

You mention Laury's rights, but even if he requests more permissions, he can't create channels and manage globally the server. How can we obtain a channel for GitHub notification or manage moderation policy ?

> Yes we can, but if you think about it for 30 seconds it's not in anyone's best interest to delete the entire channel like that, censor a discussion about Upsilon, or deprive yourself of history. We are human and have no interest in doing anything quoted here.

At the moment, yes. But by the past, we saw an organization with the same convictions as you, open-source and transparency, who make a turn-over. Yes, This organization is Numworks! How can we trust human out the contributors of our project?

> Afterwards, your opinion as a unitary person, even in a 100% Upsilon server, will not prevail. It will take some agreement between parties.

Because your opinion prevail? Actually, you won't create a real poll and discussion about this. [This citation](https://i.imgur.com/RviJU79.png) can only confirm what I say. You put this in bold and close any discussion about it. I saw that like an abuse of your permissions.
You say "democracy wanted", but when 2 mods and one admin take part of the debate, it's not a democracy. 

> In addition, as you have noticed, in the very principle of Discord roles a certain hierarchy means that not everyone has the right to delete or modify everything.

Yes sure! I can't agree with you more! 
But we choose our leader by meritocracy. Actually, no more of the Omega's leaders have the legitimacy to be our Discord's leaders! Yes, you did important things to our community, but you all left the project.

Quentinguidee, never contribute in this Upsilon, you only reply to an issue to defend your personal interest. I don't see any contribution in Omega from you since May 2021.
M4xi1m3 don't contribute to Upsilon community. And RedGl0w's last contribution to Omega is 6 months old, I didn't contribute to Upsilon too. 
How can you lead our support and communication platform without know what we did ?

So, thanks for your contributions and what you did for the community. But it's now the time to grow Upsilon at a next level and do what you did with `r/numworks` and `r/numworksomega`.

**Comment by Lauryy06** [comment]
> Quentinguidee, never contribute in this Upsilon, you only reply to an issue to defend your personal interest. I don't see any contribution in Omega from you since May 2021.
M4xi1m3 don't contribute to Upsilon community. And RedGl0w's last contribution to Omega is 6 months old, I didn't contribute to Upsilon too.

I think this is a little bit exaggerated, keep in mind that most of the features of the upsilon come from Omega.

I also think that it is not in the interest of the omega team to censor our messages, because unlike NumWorks they are not a for-profit company.

For the time being, I will not open a new Discord server. I trust the current team of moderators in the Omega community, and we have proven that we can discuss with them. Also, I don't want to waste my time managing debates like this or others by moderating a server myself. If you wish, you can see them as "partners" to whom I delegate all moderation tasks in Discord.

**Comment by FaustinM** [comment]
Thanks for your response!

Actually, I understand why you won't create a separate server. But I think about the future and newcomer, why we need to use the brand and the background of Omega ?

If I'm a newcomer, I will see Omega, I will see Quentin and the rest of Lead Team. I will see Contributors and Beta Tester. But it doesn't reflect Upsilon. 
The Contributors may pass to something else, and a lot of them pass to something else, why there are at upper than Upsilon's contributor ? 
Why there are sorts of VIP, Beta-Tester ? Upsilon never own a private beta philosophy.
Why we use the brand of Omega for Upsilon discussion ?
Why Quentin is in Core Team but not in Github Core Team ?
Why the media in join channel talk about an old project ?
Why GitHub Notification is not Upsilon Notification?
This is a server to centralize Numworks's project, but actually there is only Upsilon, the son of Omega. The only difference between them is the lead team. It's like Audacity and Tenacity. Why we need to keep the Omega brand ? It's just something to lost newcomer. Reminder we will always pass to something else at some point, and we need to explain for the newcomers, how they can help up. The human resource is critical!

Like I say before, the old team contribute a lot, and thanks for that. We use a lot of theirs contributions for our project. They create a lot of things for us. But like I say, they need to switch to something else. And I understand totally that after one year of management. We need a server that reflect our philosophy and vision! It's essential for the newcomers, and you know how recruitment of newcomer is critical with Epsilon 16 and the marketing of Numworks.

**Comment by Lauryy06** [comment]
First, for everything related to the roles on Discord, there will be a rework in a short time.

For the remnants of Omega, I guess they will fade away with time, but anyway, if someone really wants to participate in the project, I think it won't be too hard for him to understand that Omega is the "ancestor of Upsilon", now stopped.

Finally, for the moment the brand and the omega fund bring more visibility to Upsilon than they cost him, but if ever the project gets too big, creating another discord server is indeed not to be excluded.

**Comment by FaustinM** [comment]
> I think it won't be too hard for him to understand that Omega is the "ancestor of Upsilon", now stopped.

Actually, we can't see Upsilon in any welcome channel, a newcomer will only see the archive status and the disband message of Omega.

Why don't create the Discord today? We will lose all the conversations during the transitions. And like I say in my first message, we will create a community around our actual discord and when we need to do a transition, we will miss a lot of our hobbyist. 

Don't forget, we all start from scratch and something that seem obvious to you is not for someone who never join a tech community. Just look the join channel to see that a lot of people left without saying anything.

**Comment by Syycorax** [comment]
Before reading further, please note that I asked Laury's approval before posting this. This message represents only my opinion, except when "us" or "we" is used. I'm not pretending to represent neither Upsilon nor Omega (even though the 'Community Manager' role was given to me). This is only my personal reflections about your issue. Every time "us" or "we" is used, the Omega staff was consulted and agrees with my statements.

> Btw, this message explains why I can't talk without any trouble.

This message does not explain anything except @quentinguidee 's anger about your words. This doesn't have anything of a threat against you or any contributor of Upsilon.

> Who can ensure you don't censor our debate, or try to alter the reality ?

In my opinion you are the one altering the reality to match your convictions. The staff has literally no interest as stated higher to censore anything, and is standing against Numworks so that projects like Upsilon can exist. Omega took a position against Numworks and censorship, as you told with r/numworksomega. 

> You mention Laury's rights, but even if he requests more permissions, he can't create channels and manage globally the server.

Again, as Quentin said, Laury did not request permissions, except the right to pin messages, as this message state [https://imgur.com/a/CQFoCes](https://imgur.com/a/CQFoCes) which is done.

> At the moment, yes. But by the past, we saw an organization with the same convictions as you, open-source and transparency, who make a turn-over. Yes, This organization is Numworks! How can we trust human out the contributors of our project?

You are again altering the reality to match your point, whilst ignoring crucial factors such as those which led to Numworks' turnover. Omega never made financial profit, neither will it in the future. Numworks' turnover was without a doubt drove by economic interests. It is outrageous stating we could betray the community as much as Numworks. In fact, Numworks' choice of license clearly shows that they were not, since the beginning, the "open-source and transparent" organization you are describing.

> Because your opinion prevail?

@quentinguidee never stated that his opinion was prevailing on any other, just that standing against everyone and then complaining elsewhere won't lead anywhere but just make more drama.

> Actually, you won't create a real poll and discussion about this

Again, you're altering the reality at your will (I'm beginning to see a pattern here, don't you?). You started the discussion yesterday. The debate kept going until you decided to leave the server. The discussion was not censored at any moment and I think the reactions to this issue are a great representation of people's thoughts. I also have trouble finding the message where you requested a poll, and Quentin smoothed the warn message as requested by "the majority".

> I saw that like an abuse of your permissions.

This debate needed an end, no new arguments were brought neither by you nor by us.

> You say "democracy wanted", but when 2 mods and one admin take part of the debate, it's not a democracy.

I can't express how much I disagree with you on this point. You are knowingly bending reality to match your point.

You are forgetting at least :

- @Lauryy06 
- @Hyperengined
- @devdl11
- @Cyber-Thibaut
- @Yaya-Cout
- @lolocomotive

Most of them are contributors to Upsilon. Their voice are as heavy as yours.

> But we choose our leader by meritocracy. Actually, no more of the Omega's leaders have the legitimacy to be our Discord's leaders!

You are actually the first one to question the legitimacy of the Discord staff. I think it's time for Upsilon to choose a spokesperson or a "core team" so all that drama coming from people trying to represent Upsilon will not occur again.

> you only reply to an issue to defend your personal interest

I don't think the staff team is the main entity having interests. Omega has 1k people on the Discord, and 1.7k users. I think Upsilon could benefit way more from this user base than Quentin from having admin rights on the server... 

> "Quentinguidee, never contribute in this Upsilon" , "M4xi1m3 don't contribute to Upsilon community." "RedGl0w's last contribution to Omega is 6 months old, I didn't contribute to Upsilon too."

First of all, should I remind you that Upsilon is a fork of Omega? Quentin was the pioneer of the project. At this date, he wrote more than 8k lines for this project. Maxime wrote more that 6k lines for this project. Last but not least, you are ignoring the help Redglow provided and is providing for the development of Upsilon (just look at #upsilon). Not earlier than last Saturday, Redglow offered some help to Overengined, and at this date he wrote more than 1k lines of code for Omega.

> How can you lead our support and communication platform without know what we did ?

Not contributing to Upsilon doesn't mean ignoring the work Upsilon contributors are doing, which I think is awesome.

I would also like to add that, although you are trying to carry the voice of the Upsilon project, you weren't elected by anyone nor are you deserving enough to be that spokesperson, the later should be chosen, as you said, by meritocracy. Laury clearly disassociated himself and Upsilon from you: https://imgur.com/a/u7jr7re

For all the above reasons, I cannot consider your points as valid, as they are biased and deform reality with the sole purpose of matching your arguments.

**Comment by BloomyInDev** [comment]
I think that was an good answer
